apps/android/app/src/main/java/chat/simplex/app/views/chat/ComposeView.kt

```diff
@@ -118,6 +117,18 @@ data class ComposeState(
   }
 }
 
+sealed class RecordingState {
+  object NotStarted: RecordingState()
+  class Started(val filePath: String, val stopRecOnNextClick: MutableState<Boolean> = mutableStateOf(false)): RecordingState()
```

Comment:
stopRecOnNextClick seems to be a local concern of a button, why is it needed on this level?